rules-tests/Php70/Rector/FuncCall/MultiDirnameRector/Fixture/fixture.php.inc

```diff
@@ -7,16 +7,6 @@ function multiDirname()
     dirname(dirname($path));
 
     new dirname(dirname(dirname($path)));
-
-
-// untouched
```

Comment:
Moved these cases to a new file to be able to check that the rule is not actually applied

Reply:
Awesome :clap: